awx/playbooks/run_isolated.yml

```diff
@@ -18,7 +18,7 @@
         src: "{{src}}"
         dest: "{{dest}}"
 
-    - stat: path="{{src}}/env/ssh_key"
+    - local_action: stat path="{{src}}/env/ssh_key"
```

Comment:
@chrismeyersfsu this is why key copying wasn't working - we're checking to see if there *is* an SSH key to copy from the controller to the isolated node, but we were checking for file existence on the *isolated node*, **not** the controller.